e2e-tests/pages/super_search_page.py

```diff
@@ -80,26 +81,43 @@ def click_new_line(self):
 
     def select_facet(self, line_id, field):
         input_locator = (self._facet_field_locator[0], self._facet_field_locator[1] % line_id)
-        self.wait.until(lambda s: self.is_element_displayed(*self._highlighted_text_locator))
         facet_field = self.find_element(*input_locator)
-        facet_field.send_keys(field)
-        self.wait.until(lambda s: self.is_element_displayed(*self._highlighted_text_locator))
-        facet_field.send_keys(Keys.RETURN)
+        facet_field.click()
+        xpath = '//div[@class="select2-result-label" and' +\
+            ' text()[contains(.,"%s")]]' % (field)
```

Comment:
I have two problems with this change. First is that we don't use XPATH in any of the e2e tests--we use CSS selectors everywhere. Second is that this code isn't selecting the facet when run with the Firefox webdriver.

If we want this change, then we need to either convert it to a CSS selector or provide a comment about why that's impossible. If it turns out it's impossible, I suggest we instead spend some time making the HTML/JS/CSS code that makes up supersearch easier to write tests for. Right now it looks like a bunch of autogenerated mumbo-jumbo that's really hard to key off of.